src/cascadia/TerminalSettingsEditor/ColorSchemesPageViewModel.cpp

```diff
@@ -34,26 +34,36 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         // we still have the same CurrentScheme as before (if that scheme still exists)
 
         // Store the name of the current scheme
-        const auto currentSchemeName = CurrentScheme().Name();
+        const auto hasCurrentScheme = HasCurrentScheme();
+        const auto currentSchemeName = hasCurrentScheme ? CurrentScheme().Name() : hstring{ L"" };

```

Comment:
```suggestion
        const auto currentSchemeName = hasCurrentScheme ? CurrentScheme().Name() : hstring{};
```

prefer the nullary constructor instead of emitting an empty string constant